package.json

```diff
@@ -65,7 +65,7 @@
     "build_test": "shx rm -rf ./dist/ && npm-run-all build_cjs clean_spec build_spec test_mocha",
     "build_cover": "shx rm -rf ./dist/ && npm-run-all build_cjs build_spec cover",
     "build_docs": "npm-run-all build_es6_for_docs build_global build_spec tests2png decision_tree_widget && esdoc -c esdoc.json",
-    "build_spec": "tsc --project ./spec --pretty",
+    "build_spec": "tsc --project ./spec",
```

Comment:
It was to get errors to be detected by vscode, this will be restored once the branch gets ready for land.